.github/workflows/re2.yml

```diff
@@ -0,0 +1,88 @@
+name: re2
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  build:
+    runs-on: build-files-runner
```

Comment:
This should be `self-hosted` instead of `build-files-runner` because we will use self-hosted build machines.

Reply:
Resolved with 6c9525274a351b63211ba8395681cfa4591c0be2